inc/config.php

```diff
@@ -220,7 +220,7 @@
                   }
                } else if (!isset($CFG_GLPI['dbversion']) || trim($CFG_GLPI["dbversion"]) < GLPI_SCHEMA_VERSION) {
                   $older = true;
-               } else if (trim($CFG_GLPI["dbversion"]) > GLPI_SCHEMA_VERSION) {
+               } else if (!isset($CFG_GLPI['dbversion']) || trim($CFG_GLPI["dbversion"]) > GLPI_SCHEMA_VERSION) {
```

Comment:
If `isset($CFG_GLPI['dbversion'])`, the previous `else if` will be true, so this new test is not required.

Reply:
Corrected